.github/workflows/ci.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         python-version: [ '3.7', '3.8', '3.9', '3.10', '3.11' ]
         include:
           - os: ubuntu-20.04
-            python-version: '3.6'
+            python-version: '3.7'
```

Comment:
same as above, all three lines to be removed

Reply:
will do

Reply:
@mtitov : Sorry, I lost the context of this comment - what lines should be removed?